src/components/container/_container.scss

```diff
@@ -3,6 +3,9 @@
   margin: 0 auto;
   margin-bottom: 20px;
   width: 100%;
+  label p, span p {
```

Comment:
this will apply to all p tags that have parent label/span of any level.
please use the below selectors -
label > p, span > p

Reply:
done